<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtWidgetsFL</class>
 <widget class="QWidget" name="QtWidgetsFL">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>635</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>635</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>文件加载</string>
  </property>
  <widget class="QLabel" name="labelWind">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>站点及路线选择</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditStop">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditLine">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonStop">
   <property name="geometry">
    <rect>
     <x>542</x>
     <y>110</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLine">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>180</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览...</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelStop">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>站点选择</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelStop_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>线路选择</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHits">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>561</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDone">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonQuit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>230</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
